php-frontend/src/test/java/org/sonar/php/tree/impl/declaration/TypeNameTreeTest.java

```diff
@@ -38,6 +38,14 @@ public void built_in_type() throws Exception {
     assertThat(tree.token().text()).isEqualTo("int");
   }
 
+  @Test
+  public void built_in_mixed_type() throws Exception {
```

Comment:
I am not 100% sure about the impact of this test!

Reply:
I think it's good. Tests that it is correctly interpreted as a `BUILT_IN_TYPE` and not as a `NAMESPACE_NAME` (which was the case before I suppose.)